src/third_party/catch/include/helpers.hpp

```diff
@@ -103,7 +103,10 @@
     auto collection_count_with_opts = libmongoc::collection_count_with_opts.create_instance();    \
     auto collection_create_index = libmongoc::collection_create_index.create_instance();          \
     auto collection_find = libmongoc::collection_find.create_instance();                          \
-    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();
+    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();                \
+    auto collection_get_name = libmongoc::collection_get_name.create_instance();                  \
+    collection_get_name->interpose([](mongoc_collection_t*){ return "dummy_collection"; });       \
+    auto collection_rename = libmongoc::collection_rename.create_instance();
```

Comment:
No, the old test code relied on name() to work; since it now relies on a mongoc method I had to mock that.

Personally I don't think we can meaningfully test for rename(); we can check whether the basic interface was used, but the resulting code would be more of a liability than safety.

If we want to test rename() we should do it against a real db.